src/components/select-button/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import './index.css';
+
+interface SelectButtonProps {
+  label: string;
+  data: { label: string; isActive: boolean }[];
+}
+
+const SelectButton: React.FC<SelectButtonProps> = ({
+  label,
+  data,
```

Comment:
isActive ve setIsActive props kalmalı

Reply:
isActive datadan çıkarında buttonu nasıl tıklayınce active yapabilirim ?

Reply:
setlemek olacak yaptığında olmuş gibi ama eksikleri söylebilirim.
setIsActive: (data: string | null) => void;  neden data yazsın mı dizi değil ki bir item almamız için item yazabilirsin.

| null gerek yok, eğer data dizi [null] içinde null koyabilirler diye düşünüyor musun. ama gerek yok | null kaldırabilirsin zorunlu bu zaten,